ticket_booking_app/src/main/java/com/mordor/model/ReservationConfirmation.java

```diff
@@ -0,0 +1,42 @@
+package com.mordor.model;
+
+import java.net.URL;
+import java.time.Instant;
+
+public class ReservationConfirmation {
+	
+	private double totalAmount;

```

Comment:
Czemu kasa w double? W javie są dedykowane klasy do takich rzeczy. Wiesz jakie?

Reply:
To jest błąd, bo double używa reprezentacji binarnej, więc nie da się dokładnie zapisać wszystkich wartości w systemie dziesiętnym. Mogłem użyć BigDecimal, gdzie występuje podstawa 10.